src/Mautic.php

```diff
@@ -32,16 +33,24 @@ class Mautic
      */
     protected $cookie;
 
+    /**
+     * Mautic Configuration
+     *
+     * @var MauticConfig
+     */
+    protected $config;
+
     /**
      * Constructor
      *
      * @param string $baseUrl
      */
-    public function __construct($baseUrl)
+    public function __construct($baseUrl, $config)
     {
         $this->baseUrl = rtrim(trim($baseUrl), '/');
         $this->cookie = new MauticCookie;
-        $this->contact = new Contact($this->cookie);
+	$this->contact = new Contact($this->cookie);
+	$this->config = $config;
```

Comment:
```suggestion
	$this->config = $config ?: new Config;
```
If no Config DTO is set, let's create it and use the default values from it.